lib/ViewModels/FeatureInfoPanelSectionViewModel.js

```diff
@@ -29,6 +35,18 @@ var FeatureInfoPanelSectionViewModel = function(terria, feature, catalogItem) {
     this.feature = feature;
     this.name = feature.name ? feature.name : '';
     var template = defined(catalogItem) ? catalogItem.featureInfoTemplate : undefined;
+    var data = propertyValues(feature.properties, terria.clock);
+
+    knockout.track(this, ['name', 'templatedInfo', 'rawData', 'catalogItemName', 'rawDataVisible', 'downloadOptionsVisible', 'downloadDropdownPosition']);
+    knockout.track(this.terria, ['selectedFeature']);
```

Comment:
Terria is already rigging up `selectedFeature` to be observable.  I don't think this line is needed.